gruntfile.js

```diff
@@ -1,4 +1,39 @@
 module.exports = function (grunt) {
+    function _createRegEx(str) {
```

Comment:
Name is a bit misleading as it's actually encoding the str not creating a regular expression